tools/perspective-scripts/setup.mjs

```diff
@@ -168,7 +168,14 @@ async function focus_package() {
             message: "Focus NPM package(s)?",
             default: () => {
                 if (CONFIG["PACKAGE"]) {
-                    return CONFIG["PACKAGE"].split(",");
+                    const packages = CONFIG["PACKAGE"].split(",");
```

Comment:
to test this, run`pnpm -w run setup`, select perspective-pyodide. then run setup again. perspective-pyodide should be still selected in the menu.

previously the menu selected perspective-python on the second go - which made it easy to accidentally switch back to cpython